catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogPropertiesMeta.java

```diff
@@ -88,6 +92,16 @@ public class HiveCatalogPropertiesMeta extends BaseCatalogPropertiesMetadata {
               FETCH_TIMEOUT_SEC,
               PropertyEntry.integerOptionalPropertyEntry(
                   FETCH_TIMEOUT_SEC, "The timeout to fetch key tab", true, 60, false))
+            .put(
+                    LIST_ALL_TABLES,
+                    PropertyEntry.booleanPropertyEntry(
+                            LIST_ALL_TABLES,
+                            "list all tables with iceberg tables",
```

Comment:
list all tables (including the iceberg tables).

Reply:
done